### PR TITLE
Use shortcode when classic editor plugin is active

### LINKED ISSUE #4833: Use shortcode when classic editor plugin is active
## Feature Description

If a user has the classic-editor plugin active, the embed story block does not preview and seems broken. Instead of using the embed when classic-editor / TinyMCE is active, using the shortcode. See #3135 

## Additional Context
Video to show the user experience. 
https://www.youtube.com/watch?v=Id4Cr_HJoCA

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by spacedmonkey**
## Summary

If a user has the classic-editor plugin active, the embed story block does not preview and seems broken. Instead of using the embed when classic-editor / TinyMCE is active, using the shortcode. See #3135

## Testing Instructions
1. Activate classic editor plugin. 
2. Create a story.
3. Publish story. 
4. Click add post. 
5. Go to post edit screen, see shortcode.

---

Fixes #4833
Blocked #4780

### CODE REVIEW

File: includes/Admin.php [hunk 1]
```diff
@@ -91,11 +91,12 @@ public function admin_body_class( $class ) {
 	 *
 	 * @since 1.0.0
 	 *
-	 * @param string $content Default post content.
+	 * @param string   $content Default post content.
+	 * @param \WP_Post $post Post object.
```
Comment (swissspidy) [inline includes/Admin.php]: nit: can we align the params here? I thought PHPCS does enforce that.

Comment (spacedmonkey) [reply includes/Admin.php]: Seems like it is a known issue. https://github.com/squizlabs/PHP_CodeSniffer/issues/2379

Comment (spacedmonkey) [reply includes/Admin.php]: Fixed in https://github.com/google/web-stories-wp/pull/4849/commits/14c2a0e58d51b5c4da99a12e55594f484a509536

File: tests/e2e/specs/editor/publishingFlow.js [hunk 2]
```diff
@@ -90,4 +94,34 @@ describe('Publishing Flow', () => {
...
+    // Publish story.
+    await expect(page).toClick('button', { text: 'Publish' });
+
+    await expect(page).toMatchElement('button', { text: 'Dismiss' });
+
+    // Create new post and embed story.
+    await expect(page).toClick('a', { text: 'Add to new post' });
+    await page.waitForNavigation();
+
+    await expect(page).toMatch('Publishing Flow Test');
```
Comment (swissspidy) [inline tests/e2e/specs/editor/publishingFlow.js:117]: This also needs an `expect(await getEditedPostContent()).toMatchSnapshot();` like the above test to actually verify that the shortcode is in the content.

Comment (swissspidy) [reply tests/e2e/specs/editor/publishingFlow.js:117]: Oh, wait, that's a Gutenberg-only method though. Needs some other way to find the post content.

Comment (spacedmonkey) [reply tests/e2e/specs/editor/publishingFlow.js:117]: I removed this for a reason. There isn't a way easily get text content from the tinymce editor. Isn't the render shotcode the front end enough.

**Comment by spacedmonkey** [comment]
This PR is now blocked by #4780 as the e2e test fails without it.

**Comment by spacedmonkey** [comment]
This PR is ready go to now that #3135 is merged.

File: tests/e2e/specs/editor/publishingFlow.js (hunk 2 above)
Comment (swissspidy) [reply tests/e2e/specs/editor/publishingFlow.js:117]: It's as simple as `document.getElementById('content').innerHTML` or `document.querySelector('.wp-editor-area').innerHTML` if that's preferable

**Review by swissspidy — APPROVED** [review]
LGTM with one nit

File: tests/e2e/specs/editor/publishingFlow.js (hunk 2 above)
Comment (spacedmonkey) [reply tests/e2e/specs/editor/publishingFlow.js:117]: Added a fix here - https://github.com/google/web-stories-wp/pull/4849/commits/89a65e8d2fe10a83131cfc02626b5733561419cc

**Comment by csossi** [comment]
Verified in QA

**Pull request merged**